src/main/java/com/khutircraftubackend/product/response/ProductResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.khutircraftubackend.product.response;
+
+import com.khutircraftubackend.category.CategoryEntity;
+import com.khutircraftubackend.seller.SellerEntity;
+import lombok.Builder;
+
+@Builder
+public record ProductResponse(
+        Long id,
+        String name,
+        String thumbnailImageUrl,
+        String imageUrl,
+        boolean available,
+        String description,
+        SellerEntity seller,
+        CategoryEntity category
```

Comment:
will these 2 guys be mapped to IDs (not objects) ?

Reply:
I planned to return the object itself so that it would be possible to extract the necessary data from response.